.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 server.pot
+.DS_Store
```

Comment:
Do not add this to .gitignore.

Use a global gitignore for your machine:
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore